### DISCUSSION TITLE
Feature: Site Notice (global header with Markdown)

### DISCUSSION THREAD

**Discussion by shield-9** [growilabs/growi#7710] [Ideas / Requests]
I'd like to display a site-wide notice at the top of each page. (like [Sitenotice of Anonnotice in MediaWiki](https://www.mediawiki.org/wiki/Manual:Interface/Sitenotice))

This feature is very useful to announce about maintenance and etc.

**Comment by yuki-takei** [comment]
@shield-9 Thank you for your post.

I can't imagine that feature because I don't have enough experience to use MediaWiki.
Would you describe more details or post rough screen mock?

**Comment by shield-9** [comment]
Japanese Wikipedia is [currently using this feature](https://ja.wikipedia.org/wiki/MediaWiki:Sitenotice) (MediaWiki): 
![MediaWiki Sitenotice   Wikipedia](https://user-images.githubusercontent.com/3660715/59146698-5cb3eb80-8a2c-11e9-8f4c-f6de6e12104c.png)

And this is rough mock for GROWI:
![GroupWiki](https://user-images.githubusercontent.com/3660715/59146534-25443f80-8a2a-11e9-92c6-0fba26bbce42.png)

**Comment by yuki-takei** [comment]
@shield-9 Thank you. Now I understand what you would like to.
We'll consider it.
